include/triton/common/logging.h

```diff
@@ -73,9 +74,42 @@ class Logger {
   // Get the logging format.
   Format LogFormat() { return format_; }
 
+  // Get the logging format as a string.
+  std::string LogFormatString()
+  {
+    switch (format_) {
+      case Format::kISO8601:
+        return "ISO8601";
+      case Format::kDEFAULT:
+        return "default";
+      default:
+        return "";
+    }
+  }
+
   // Set the logging format.
   void SetLogFormat(Format format) { format_ = format; }
 
+  // Get the log output file name.
+  const std::string& LogFile() { return filename_; }
+
+  // Set the log output file.
+  bool SetLogFile(const std::string& filename)
+  {
+    const std::lock_guard<std::mutex> lock(mutex_);
+    file_stream_.close();
+    filename_ = filename;
+    if(!filename.empty()) {
+      file_stream_.open(filename_, std::ios::app);
```

Comment:
Handle exception

Reply:
Error is handled below. Will revert to the previous log file and return false. Return status will be handled on the server side.